src/consensus/aft/raft.h

```diff
@@ -779,6 +786,10 @@ namespace aft
               state->view_history.update(state->commit_idx + 1, sig_term);
               commit_if_possible(r.leader_commit_idx);
             }
+            if (consensus_type == ConsensusType::BFT)
+            {
+              send_append_entries_signed_response(r.from_node, sig);
```

Comment:
This is probably something to discuss later, but we could have a config level where those signatures are produced, sent and recorded just for offline audit but without gating the commit level of the consensus.